README.md

```diff
@@ -13,32 +13,59 @@
 
 ---
 
-Bodywork deploys machine learning projects developed in Python, to Kubernetes. It helps you to:
+Bodywork is a command line tool that deploys machine learning pipelines to [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes). It takes care of everything to do with containers and orchestration, so that you don't have to.
 
-* serve models as microservices
-* execute batch jobs
-* run reproducible pipelines
+## Who is this for?
 
-On demand, or on a schedule. It automates repetitive DevOps tasks and frees machine learning engineers to focus on what they do best - solving data problems with machine learning.
+Bodywork is aimed at teams who want a solution for running ML pipelines and deploying models to Kubernetes. It is a lightweight and simpler alternative to [Kubeflow](https://www.kubeflow.org), or to building your own platform based around a workflow orchestration tool like [Apache Airflow](https://airflow.apache.org), [Argo Workflows](https://argoproj.github.io/workflows/) or [Dagster](https://www.dagster.io).
+
+## Pipeline = Jobs + Services
+
+Any stage in a Bodywork pipeline can do one of two things:
+
+- [x] **run a batch job**, to prepare features, train models, compute batch predictions, etc.
+- [x] **start a long-running process**, like a Flask app that serves model predictions via HTTP.
+
+You can use these to compose pipelines for many common ML use-cases, from serving pre-trained models to running continuous training on a schedule.
+
+## No Boilerplate Code Required
+
+Defining a stage is as simple as developing an executable Python module or Jupyter notebook that performs the required tasks, and then committing it to your project's Git repository. You are free to structure your codebase as you wish and there are no new APIs to learn.
+
+<div align="center">
+<img src="https://bodywork-media.s3.eu-west-2.amazonaws.com/project_structure_map.png"/ alt="Git project structure">
+</div>
+
+## Easy to Configure
+
+Stages are assembled into [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph) that define your pipeline's workflow. This and other key [configuration](user_guide.md#configuring-a-project-for-deployment-with-bodywork) is contained within a single [bodywork.yaml file](https://github.com/bodywork-ml/bodywork-ml-pipeline-project/blob/master/bodywork.yaml).
+
+## Simplified DevOps for ML
+
+Bodywork removes the need for you to build and manage container images for any stage of your pipeline. It works by running all stages using Bodywork's [custom container image](https://hub.docker.com/repository/docker/bodyworkml/bodywork-core). Each stage starts by pulling all the required files directly from your project's Git repository (e.g., from GitHub), pip-installing any required dependencies, and then running the stage's designated Python module (or Jupyter notebook).
+
+<div align="center">
+<img src="https://bodywork-media.s3.eu-west-2.amazonaws.com/ml_pipeline.png"/>
+</div>
```

Comment:
I think we should add a line in somewhere just saying that it works on any K8s platform, no matter where it's hosted to be clear. This is one of our important features and likely something someone will want to know from the off.